include/linux/medusa/l3/constants.h

```diff
@@ -30,14 +30,14 @@
 #define MEDUSA_ACCNAME_MAX	MEDUSA_EVNAME_MAX
 #define MEDUSA_SERVERNAME_MAX	128
 
-/* answer codes */
+#define not_supported ("do not use - behavior not supported by LSM")
 
 typedef enum {
-	MED_ERR =	-1,	/* error */
-	MED_YES =	 0,	/* permit the operation */
-	MED_NO =	 1,	/* forbid the operation */
-	MED_SKIP =	 2,	/* forbid the operation, but return success */
-	MED_OK =	 3	/* permit the operation, but proceed with
+	MED_ERR =		-1,	/* error */
+	MED_FORCE_ALLOW __attribute__ ((deprecated(not_supported))),	/* permit the operation */
+	MED_DENY =		 1,	/* forbid the operation */
+	MED_FAKE_ALLOW __attribute__ ((deprecated(not_supported))),	/* forbid the operation, but return success */
```

Comment:
we  need MED_FORCE_ALLOW and MED_FAKE_ALLOW in L4 production and development converters, so I'm not sure about deprecating them

Reply:
Did not we discuss this previously? MED_SKIP and MED_YES cannot be used right now because of the restrictions of LSM so thats why we are dropping support for now.

There is no reason in having the support if we cannot use it, allowing it will just create a risk of misuse of codes, the thing I was trying to solve with this PR. i thought it was clear from our communication, no?

Reply:
After the discussion through emails, it was decided to not deprecate the results due to upcoming refactorings.